File: modules/Volo/modules/Volo.Chat/src/Volo.Chat.Application.Contracts/Volo/Chat/Conversations/ChatConversationDto.cs
using System.Collections.Generic;
using Volo.Chat.Messages;
using Volo.Chat.Users;

namespace Volo.Chat.Conversations
{
    public class ChatConversationDto
    {
        public List<ChatMessageDto> Messages { get; set; }

        public ChatTargetUserInfo TargetUserInfo { get; set; }
    }
}

File: Volo.Abp/Volo.Chat/angular/projects/chat/src/lib/models/chat-conversation-dto.ts
import { ChatTargetUserInfo } from './chat-target-user-info';
import { ChatMessageDto } from './chat-message-dto';
export class ChatConversationDto {
  messages: ChatMessageDto[];
  targetUserInfo: ChatTargetUserInfo;

  constructor(initialValues: Partial<ChatConversationDto> = {}) {
    if (initialValues) {
      for (const key in initialValues) {
        if (initialValues.hasOwnProperty(key)) {
          this[key] = initialValues[key];
        }
      }
    }
  }
}
